### PR TITLE
[GUI] Extract the type casts of environment variables in misc.gui.gui into a reusable function

### PR DESCRIPTION
**Pull request by jinmu333**
[GUI] move the type cast of the environment variable in misc.gui.gui to a function

### CODE REVIEW

File: python/taichi/misc/gui.py [hunk 1]
```diff
@@ -120,6 +117,18 @@ def value(self):
...
             self.gui.core.set_widget_value(self.wid, value)
 
+    def get_bool_environ(self, key, default):
+        """Get an environment variable and cast to bool,
+            return default if it doesn't exist.
+        Args:
+            key (str): The environment variable key.
+            default (bool): The default value.
+        Return: bool
+        """
```
Comment (yuanming-hu) [inline python/taichi/misc/gui.py:127]:
```suggestion
        """Get an environment variable and cast to bool.
        Args:
            key (str): The environment variable key.
            default (bool): The default value.
        Return:
            The environment variable value cast to bool. If the value is not found, directly return argument 'default'.
        """
```

**Review by yuanming-hu — COMMENTED** [review]
Welcome to the Taichi community!

I'm not a Python docstring expert, but maybe this comment makes the docstring a little clearer.

**Comment by jinmu333** [comment]
> Welcome to the Taichi community!
> 
> I'm not a Python docstring expert, but maybe this comment makes the docstring a little clearer.

Thanks，I have updated the document of this function

**Comment by yuanming-hu** [comment]
Hi @jinmu333, in order to get this PR merged, would you mind...
1. Signing the [CLA](https://cla-assistant.io/taichi-dev/taichi?pullRequest=3065);
2. Marking this PR as "ready for review"?

Thank you in advance! I can merge this PR as soon as you finish the two tasks above.
(Of course, if you want to include more stuff in this PR and you'd like to keep this PR in a *draft* state, please feel free to do that!)

**Comment by jinmu333** [comment]
> finish the two tasks

Thanks, I have finish the two tasks.

**Review by yuanming-hu — APPROVED** [review]
Thank you! Everything LGTM now!

**Pull request merged**